src/ui/public/index_patterns/_get_template_index_patterns.js

```diff
@@ -0,0 +1,21 @@
+import { map, flatten, pluck } from 'lodash';
+
+const getIndexPatternsFromResponse = json => {
+  return map(flatten(pluck(json, 'index_patterns')));
```

Comment:
When setting up the above scenario, I started by naming my index templates `jim-template-1`, `jim-template-2` and `jim-template-3`, each with an `index_pattern` value of 'logstash-*'. Then in the searchbox, I typed 'log' expecting to see my duplicates pop up. Should the search text be applied to the _name_ of the index templates, or the `index_pattern` values?